SQL_module/SubjectForm/subjectForm.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
+    <title> Form task </title>
+</head>
+
+<body>
+
+    <form method="post" action="./index.php">
+        <label for="Subject"> Име на предмета: </label>
+        <input type="text" id="Subject title" name="title" require>
```

Comment:
```suggestion
        <input type="text" id="Subject title" name="title" required>
```
Typo. Please fix everywhere. Source: https://www.w3schools.com/tags/att_input_required.asp